src/zkevm_specs/evm/execution/call_staticcall.py

```diff
@@ -1,37 +1,47 @@
 from ...util import (
-    FQ,
-    N_BYTES_ACCOUNT_ADDRESS,
-    N_BYTES_GAS,
     EMPTY_CODE_HASH,
-    GAS_COST_WARM_ACCESS,
+    FQ,
     GAS_COST_ACCOUNT_COLD_ACCESS,
-    GAS_COST_NEW_ACCOUNT,
     GAS_COST_CALL_WITH_VALUE,
+    GAS_COST_NEW_ACCOUNT,
+    GAS_COST_WARM_ACCESS,
     GAS_STIPEND_CALL_WITH_VALUE,
+    N_BYTES_ACCOUNT_ADDRESS,
+    N_BYTES_GAS,
+    RLC,
 )
 from ..instruction import Instruction, Transition
+from ..opcode import Opcode
 from ..table import RW, CallContextFieldTag, AccountFieldTag
 from ..precompiled import PrecompiledAddress
 
 
-def call(instruction: Instruction):
-    instruction.responsible_opcode_lookup(instruction.opcode_lookup(True))
+def call_staticcall(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
+    is_call, _ = instruction.pair_select(opcode, Opcode.CALL, Opcode.STATICCALL)
+    instruction.responsible_opcode_lookup(opcode)
 
     callee_call_id = instruction.curr.rw_counter
 
     tx_id = instruction.call_context_lookup(CallContextFieldTag.TxId)
     reversion_info = instruction.reversion_info()
     caller_address = instruction.call_context_lookup(CallContextFieldTag.CalleeAddress)
-    is_static = instruction.call_context_lookup(CallContextFieldTag.IsStatic)
+    is_static = instruction.select(
+        is_call, instruction.call_context_lookup(CallContextFieldTag.IsStatic), FQ(1)
+    )
     depth = instruction.call_context_lookup(CallContextFieldTag.Depth)
 
+    # Verify is_static == 1 for opcode STATICCALL.
+    instruction.constrain_zero((1 - is_call) * (1 - is_static.expr()))
```

Comment:
I think this is not necessary, since in line 21 `instruction.pair_select` already constrains so (see also circuit implementation https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/main/zkevm-circuits/src/evm_circuit/util/math_gadget.rs#L590-L606).

```suggestion
```